### DISCUSSION TITLE
Async workflow

### DISCUSSION THREAD

**Discussion by berkayk** [KyryloKuzyk/PrimeTween#45] [General] [answered]
Hi, first of all, thanks for the amazing library! Everything is well thought out.

I am migrating from DOTween but have some problems migrating async methods.

Is there a way to return a tween like `return Task.CompletedTask`? 

With DoTween, I was returning like `return ...AsyncWaitForCompletion()` or `return Task.CompletedTask`

````
public Tween MoveTowards(Vector3 targetPos, float duration = DurationMove) {
      if (Vector3.Distance(transform.position, targetPos) < 0.005f) {
           return Tween.CompletedTween; //  <---- What I need :)
      }
      IsMoving = true;
      return Tween.Position(transform, targetPos, duration: duration, ease: PrimeTween.Ease.OutSine).OnComplete(target: this, target => target.OnMoveComplete());
}
````

I know there are workarounds like checking the position before calling this method but I have many methods like this.

**Comment by KyryloKuzyk** [comment, marked as answer]
I’m glad you’re enjoying PrimeTween!

I assume you’re looking for a way to return an “empty” or “dead” tween, right? Then you can simply write ‘return default;’ or ‘return new Tween();’, which are identical operations. 

You can safely await dead tweens, but it’s not allowed to add dead tweens to a sequence.

**Reply by berkayk** [reply]
Thanks 👍
